docs/gettingstarted/structuring.md

```diff
@@ -14,49 +14,65 @@ Pick a unique package name. If you own a URL associated with your project, you c
-|         name |  string  | required | The user-friendly name of this mod. |
-|  description |  string  |   `""`   | A description of this mod in 1-2 paragraphs. |
-|      version |  string  |   `""`   | The version of the mod. |
-|    mcversion |  string  |   `""`   | The Minecraft version. |
-|          url |  string  |   `""`   | A link to the mod's homepage. |
-|    updateUrl |  string  |   `""`   | Defined but unused. Superseded by updateJSON. |
-|   updateJSON |  string  |   `""`   | The URL to a [version JSON](autoupdate#forge-update-checker). |
-|   authorList | [string] |   `[]`   | A list of authors to this mod. |
-|      credits |  string  |   `""`   | A string that contains any acknowledgements you want to mention. |
+|        modid |  string  | mandatory | The modid this file is linked to. |
+|      version |  string  | mandatory | The version of the mod. It should be just numbers seperated by dots, ideally conforming to [Semantic Versioning](https://semver.org/). |
+|  displayName |  string  | mandatory | The user-friendly name of this mod. |
+| updateJSONURL |  string  |   `""`   | The URL to a [version JSON](autoupdate#forge-update-checker). |
+|   displayURL |  string  |   `""`   | A link to the mod's homepage. |
 |     logoFile |  string  |   `""`   | The path to the mod's logo. It is resolved on top of the classpath, so you should put it in a location where the name will not conflict, maybe under your own assets folder. |
-|  screenshots | [string] |   `[]`   | A list of images to be shown on the info page. Currently unimplemented. |
-|       parent |  string  |   `""`   | The modid of a parent mod, if applicable. Using this allows modules of another mod to be listed under it in the info page, like BuildCraft. |
-| useDependencyInformation |  boolean |  `false` | If true and `Mod.useMetadata`, the below 3 lists of dependencies will be used. If not, they do nothing. |
-| requiredMods | [string] |   `[]`   | A list of modids. If one is missing, the game will crash. This **does not affect the _ordering_ of mod loading!** To specify ordering as well as requirement, have a coupled entry in `dependencies`. |
-| dependencies | [string] |   `[]`   | A list of modids. All of the listed mods will load *before* this one. If one is not present, nothing happens. |
-|   dependants | [string] |   `[]`   | A list of modids. All of the listed mods will load *after* this one. If one is not present, nothing happens. |
+|      credits |  string  |   `""`   | A string that contains any acknowledgements you want to mention. |
+|      authors |  string  |   `""`   | The authors to this mod. |
+|  description |  string  | mandatory | A description of this mod. |
+| dependencies | [list] |   `[]`   | A list of dependencies of this mod. |
 
-A good example `mcmod.info` that uses many of these properties is [BuildCraft](https://gist.github.com/anonymous/05ad9a1e0220bbdc25caed89ef0a22d2).
```

Comment:
Maybe link to Forge’s mods.toml? And/Or the MDK’s one?
Shameless plug but if you need another example there’s the one in my example mod [https://github.com/Cadiboo/Example-Mod/blob/1.15.1/src/main/resources/META-INF/mods.toml](https://github.com/Cadiboo/Example-Mod/blob/1.15.1/src/main/resources/META-INF/mods.toml)

Reply:
I decided not to link one because the example mod has quite a clear file. Furthermore the old Buildcraft mcmod.info had a lot of things in it, having multiple mods and all which made it more interesting to look at.

Reply:
Forges test mod mods.toml has multiple mods https://github.com/MinecraftForge/MinecraftForge/tree/1.15.x/src/test/resources/META-INF

Reply:
I'm talking more about if it's a good example. The original buildcraft mcmod.info was good because "uses many of these properties" but with the new mods.toml the default file in the MDK is already really informative.